sympy/calculus/util.py

```diff
@@ -112,3 +117,316 @@ def elm_domain(expr, intrvl):
                                     for element in finite_set])
             _domain = Union(_domain, _domain_element)
         return _domain
+
+
+class Limits(Interval, AtomicExpr):
+    """
+    Limits represents an interval `[a, b]` that is an arbitary real number
+    greater than equal to `a` and less than equal to `b` where `a` and `b`
+    are real numbers and finite.
+
+    `<a, b> = \{x \in \mathbb{R} \,|\, a \le x \le b\}`
+
+    `<-oo, b> = \{x \in \mathbb{R} \,|\, x \le b\}`
+
+    `<a, -oo> = \{x \in \mathbb{R} \,|\, a \le x}`
+
+    Operation on intervals are defined as:
+
+    `<a, b> + <c, d> = \{ x+y| a\le x \le b, c \le y \le d}`
+
+    `<a, b> - <c, d> = \{ x-y| a\le x \le b, c \le y \le d}`
+
+    `<a, b> * <c, d> = \{ x*y | a \le x \le b, c \le y \le d\}`
+
+    There is, however, no consensus on interval division.
+
+    `<a, b> / <c, d> = \{ x/y | a \le x \le b, c \le y \le d\}`
+
+    `<a, b>^n = \{ x^n | a \le x \le b}`
+
+    Note
+    ====
+
+    The main focus in the interval arithmetic is on the simplest way to calculate
```

Comment:
@asmeurer I had added this note for what `Limits` is supposed to do and what it does not do.